engine/runtime/src/main/java/org/enso/interpreter/runtime/EnsoContext.java

```diff
@@ -381,6 +384,27 @@ public Optional<Module> findModuleByExpressionId(UUID expressionId) {
         .findFirst();
   }
 
+  /**
+   * Modifies the classpath to use to lookup {@code polyglot java} imports.
+   * @param file the file to register
+   */
+  @TruffleBoundary
+  public void addToHostClassPath(TruffleFile file) {
```

Comment:
```suggestion
  public void addClassPath(TruffleFile file) {
```

Shouldn't we rename this method?

Looking at the body, it seems that it will add either to _host_ or _guest_ Java classpath. But the method name suggests otherwise.

Reply:
Fair comment: b3332c4ed1aef714c3ae24ee1e4c4a0e35aff9ba